src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/IWcfDuplexService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ServiceModel;
+
+namespace WcfService
+{
+    [ServiceContract (CallbackContract = typeof(IWcfDuplexServiceCallback))]
+    public interface IWcfDuplexService
+    {
+        [OperationContract]
+        void Ping();
+    }
```

Comment:
Can we make Ping and OnPingCallback take a parameter, and let Ping echo it back to the callback? The test can ping a GUID and check it gets it back.  Matching up the parameters of the callback operations is an important path to test of the client layer.